server/src/main/java/org/elasticsearch/snapshots/SnapshotShardFailure.java

```diff
@@ -100,7 +114,7 @@ public String reason() {
     /**
      * Returns REST status corresponding to this failure
      *
-     * @return REST status
+     * @return REST STATUS
```

Comment:
was this necessary maybe you rather want to link to RestStatus ?

Reply:
No I think that was just an issue with a cherry-pick merge.